app/javascript/packs/application.js

```diff
@@ -52,8 +56,12 @@ Vue.component('CreateRecipes', CreateRecipes);
 Vue.component('MenusContainer', MenusContainer);
 Vue.component('NewMenuContainer', NewMenuContainer);
 
+
+Vue.component('ProviderIndexContainer', ProviderIndexContainer);
+
 Vue.filter('currency', formatCurrency);
 
+
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>More than 1 blank line not allowed  no-multiple-empty-lines